src/main/java/org/thoughtslive/jenkins/plugins/jira/steps/EditComponentStep.java

```diff
@@ -29,6 +30,10 @@
   @Getter
   private final Component component;
 
+  @Getter
+  @DataBoundSetter
+  private boolean addMetaInDescription = true;
```

Comment:
I would move this to super class [BasicJiraStep](https://github.com/jenkinsci/jira-steps-plugin/blob/master/src/main/java/org/thoughtslive/jenkins/plugins/jira/steps/BasicJiraStep.java), so that this common across all the steps, though we aren't adding meta or it is not applicable to few steps, I feel adding this to super class doesn't harm.